SPECS/ruby/ruby.spec

```diff
@@ -4,11 +4,14 @@
 %global gem_dir %{_datadir}/ruby/gems
 %global rubygems_version        3.1.6
```

Comment:
There seems to be a duplicate of this macro. Please remove the one with the lower version, thanks

Reply:
Fixed